web/views.py

```
@@ -6469,6 +6473,127 @@
+                )
+
+            messages.success(request, "Your PDF document has been submitted successfully!")
+            return redirect("pdf_submission_list")
+    else:
+        form = PDFSubmissionForm()
+
+    return render(request, "web/pdf/upload_submission.html", {"form": form})
+
+
+@login_required
+def pdf_submission_list(request):
```

Ruff (0.8.2)

Missing return type annotation for public function `pdf_submission_list`

(ANN201)